packages/server/modules/emails/services/emailRendering.ts

```diff
@@ -113,3 +114,12 @@ const renderEmailText = async (
   )
   return fullText
 }
+
+/**
+ * Sanitize message that potentially has HTML in it
+ */
+export function sanitizeMessage(message: string, stripAll: boolean = false): string {
```

Comment:
unexpected change, did something change with emails?

Reply:
Nah, this isn't anything new, it was moved from one of the deleted files